superset/db_engine_specs/base.py

```diff
@@ -901,6 +901,15 @@ def _extract_error_message(cls, ex: Exception) -> str:
         """Extract error message for queries"""
         return utils.error_msg_from_exception(ex)
 
+    @classmethod
+    def _reformat_error_message(cls, message: str) -> Tuple[str, str]:
```

Comment:
I'm not big on a function that returns a tuple, short we have an object like `DatabaseMessage`, maybe it's just a named tuple or similar. That object may have more than two props. Maybe is has (engine, error_code, message_title, short_message, details) (?)